packages/loader/driver-definitions/src/driverError.ts

```diff
@@ -94,7 +94,7 @@ export interface IGenericNetworkError extends IDriverErrorBase {
 
 export interface IAuthorizationError extends IDriverErrorBase {
     readonly errorType: DriverErrorType.authorizationError;
-    readonly claims?: string;
+    readonly claims?: { tag: string, value: string | undefined }; // See ITaggedTelemetryPropertyType
```

Comment:
This seems a bit weird. I.e. we are conflicting here the facts that it's interface for passing errors and particular implementation of this interface that also can be used to provide extra data to logger.

```typescript
export interface IAuthorizationError extends IDriverErrorBase {
    readonly errorType: DriverErrorType.authorizationError;
    readonly claims?: string;
}

export class AuthorizationError extends LoggingError implements IAuthorizationError {
   private _claims: ITaggedTelemetryPropertyType; // this is what is logged to telemetry
   get claims() { return this._claims.value; } 
}
```
I do not like _claims in telemetry, but maybe we can find a pair of names (for telemetry and for property on object) here that makes sense?

Reply:
Yeah this is a real conundrum I've been thinking about, let's discuss when I'm back online in June :)

Reply:
New plan:  https://github.com/microsoft/FluidFramework/issues/6485

Reply:
Actually:  #6900